app/views/admin/statistics_announcements/_form.html.erb

```diff
@@ -5,6 +5,7 @@
     heading_size: "l",
     name: "statistics_announcement[publication_type_id]",
     id: "statistics_announcement_publication_type_id",
+    hint: statistics_announcement.new_record? ? nil : "Please note that changing the statistics type will also automatically update the type of the connected document.",
```

Comment:
No please: "Changing the statistics type will automatically update the type of the connected document."

Reply:
I guess we're waiting for content design on this one